src/app/views/merch/merch.component.html

```diff
@@ -0,0 +1,140 @@
+<div>
+  <div id="promo">
+    <button (click)="buynow(grid)" clickable>Buy Today!</button>
+    <video autoplay muted loop>
+      <source src="app/../assets/merch/merch.mp4" />
+    </video>
```

Comment:
```suggestion
    <video autoplay muted loop>
      <source src="app/../assets/merch/merch.mp4" />
    </video>
    <button (click)="buynow(grid)" clickable>Buy Today!</button>
```

The button was not responsive for me. Maybe swap the order of the `<video>` and `<button>` elements instead, so the button is in the front.